### DISCUSSION TITLE
How do I provide a prop with spaces in a string?

### DISCUSSION THREAD

**Discussion by gbrandt** [zauberzeug/nicegui#97] [Q&A] [answered]
I am using ui.input() and it allows a prop called 'error-message'.

`self.input_add_user_email = ui.input(label='Email', on_change=self.on_change_email).props('error-message=Invalid Email').style('width: 300px')`

However only the text `Invalid` is shown on the error when I do this:  `self.button_add_user.props('disable=false')`

Is there anyway to get a string with spaces into the error message?

**Comment by rodja** [comment]
Thanks for pointing out this issue. It's indeed a current limitation. I created the issue #98 to fix this.

**Comment by rodja** [comment, marked as answer]
Released in [0.9.11](https://github.com/zauberzeug/nicegui/releases/tag/v0.9.11)
